lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -111,6 +111,10 @@ function findDifference(path, actual, expected) {
     return null;
   }
 
+  if (expected instanceof RegExp && expected.test(actual)) {
```

Comment:
> Really? :P

well `expected instanceof RegExp` won't work if through some ridiculously circuitous process you're writing expectations values with a `RegExp` object from another realm, so the "real" way to check is like the old `isArray` check of `Object.prototype.toString.call(expected) === '[object RegExp]'`.

I would have thought someone, somewhere would have proposed `RegExp.isRegExp()` like `Array.isArray()` to TC39 at some point, but I couldn't find any record of such a thing :)